src/main/java/sirius/kernel/health/Exceptions.java

```diff
@@ -114,6 +114,8 @@ protected ErrorHandler(boolean processError) {
 
         /**
          * Specifies which exception leaded to the error being handled.
+         * <p>
+         * Note: When used, an additional '- %s (%s)' has to be added at the end of the error message provided when concatenating a {@link ErrorHandler#withSystemErrorMessage(String, Object...)} call.
```

Comment:
Better add this to the withSystemErrorMessage call? Also, outputting the additional parameters is optional.